CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.8)
 project(steering_functions)
 
 set(PROJECT_VERSION 0.2.0)
```

Comment:
Do we increment the project version? Is this the same version as in the package.xml?

Reply:
The ROS package version (package.xml) and the CMake project version should match yes. And we should create a tag on master when this is merged in (which we forgot for version 0.2.0).

Reply:
The CMake project version is for users who use this as a standalone library, so it should not be removed.

Reply:
Fine, let's keep it